src/store/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import update from '../reducers/datapackageReducer'
+
+export default function configureStore(preloadedState) {
+  return createStore(
+    update,
```

Comment:
Yes, I agree with that, but could we leave it for later as it is sort of improvement that is not required right now?